bugsnag/utils.py

```diff
@@ -18,9 +18,13 @@ class SanitizingJSONEncoder(JSONEncoder):
     A JSON encoder which handles filtering and conversion from JSON-
     incompatible types to strings.
 
-    >>> encoder = SanitizingJSONEncoder(filters=['bananas'])
-    >>> encoder.encode({'carrots': 4, 'bananas': 5})
-    "{'carrots': 4, 'bananas': '[FILTERED]'}"
+    >>> from json import loads
+    >>> encoder = SanitizingJSONEncoder(keyword_filters=['bananas'])
+    >>> items = loads(encoder.encode(FilterDict({'carrots': 4, 'bananas': 5})))
+    >>> items['bananas']
+    '[FILTERED]'
+    >>> items['carrots']
+    4
```

Comment:
Changed this one to avoid a scenario where output differs due to key ordering in different python versions